packages/office-ui-fabric-react/src/components/Image/Image.styles.ts

```diff
@@ -77,28 +82,43 @@ export const getStyles = (props: IImageStyleProps): IImageStyles => {
       isCenter && [classNames.imageCenter, ImageFitStyles],
       isContain && [
         classNames.imageContain,
-        isLandscape && {
-          width: '100%',
-          height: 'auto'
-        },
-        !isLandscape && {
-          width: 'auto',
-          height: '100%'
-        },
-        ImageFitStyles
-      ],
-      isCover && [
-        classNames.imageCover,
-        isLandscape && {
-          width: 'auto',
-          height: '100%'
-        },
-        !isLandscape && {
+        supportsObjectFit && {
           width: '100%',
-          height: 'auto'
+          height: '100%',
+          objectFit: 'contain'
         },
+        !supportsObjectFit &&
+          isLandscape && {
+            width: '100%',
+            height: 'auto'
+          },
+        !supportsObjectFit &&
+          !isLandscape && {
+            width: 'auto',
+            height: '100%'
+          },
         ImageFitStyles
       ],
+      isCover &&
+        isCover && [
+          classNames.imageCover,
+          supportsObjectFit && {
+            width: '100%',
+            height: '100%',
+            objectFit: 'cover'
+          },
+          !supportsObjectFit &&
+            isLandscape && {
+              width: 'auto',
+              height: '100%'
+            },
+          !supportsObjectFit &&
+            !isLandscape && {
+              width: '100%',
+              height: 'auto'
+            },
```

Comment:
Perhaps a helper for this would be nice, we could reuse the object for landscape and portrait.

Reply:
What about a fallback object ?
```typescript
const fallbackObjectFitStyles =
  ((isContain && isLandscape) || (isCover && !isLandscape)) ? { width: '100%', height: 'auto' } : { width: 'auto', height: '100%' };